rudder-lang/src/bin/main.rs

```diff
@@ -75,32 +89,47 @@ struct Opt {
     #[structopt(long, short)]
     input: PathBuf,
     /// Set to use technique translation mode
-    #[structopt(long)]
+    #[structopt(long, short)]
     translate: bool,
     /// Set to compile a single technique
-    #[structopt(long)]
-    technique: bool,
-    /// Output format to use
-    #[structopt(long, short = "f")]
-    output_format: Option<String>,
+    #[structopt(long, short)]
+    compile: bool,
+    /// Set to change default env logger behavior (INFO, DEBUG, ERROR)
+    #[structopt(long, short, default_value = "warn")]
+    logger: String,
```

Comment:
Ideally this should be a specific type (like the `Level` type from `log`), as it would prevent accepting unknown levels

Reply:
Yes ! Fixup incoming